crates/cdk-common/src/database/mint/test/proofs.rs

```diff
@@ -162,3 +162,457 @@ where
         "Duplicate entry"
     );
 }
+
+/// Test updating proofs states
+pub async fn update_proofs_states<DB>(db: DB)
```

Comment:
Do we still check for valid state transitions in db or we expect that in application